go-controller/pkg/cluster/gateway_shared_intf.go

```diff
@@ -321,21 +299,18 @@ func initSharedGateway(
 	}
 
 	err = util.GatewayInit(clusterIPSubnet, nodeName, ifaceID, ipAddress,
-		macAddress, gwNextHop, subnet, true, lspArgs)
+		macAddress, gwNextHop, subnet, util.PhysicalNetworkName, lspArgs)
 	if err != nil {
 		return fmt.Errorf("failed to init shared interface gateway: %v", err)
 	}
 
-	// Add static routes to OVN Cluster Router to enable pods on this Node to
-	// reach the host IP
-	err = addStaticRouteToHost(nodeName, ipAddress)
-	if err != nil {
+	// Program cluster.GatewayIntf to let non-pod traffic to go to host
+	// stack
+	if err = addDefaultConntrackRules(nodeName, bridgeName, gwIntf); err != nil {
 		return err
 	}
 
-	// Program cluster.GatewayIntf to let non-pod traffic to go to host
-	// stack
-	if err := addDefaultConntrackRules(nodeName, bridgeName, gwIntf); err != nil {
+	if err = initLocalOnlyGateway(nodeName, clusterIPSubnet); err != nil {
 		return err
 	}
```

Comment:
May be we should move initLocalOnlyGateway() to immediately after GatewayInit()?